exercises/complete/aws-kms-key-arn/2.5.toml

```diff
@@ -1,4 +1,4 @@
-target = "aws-kms-key-arn.txt#2.5"
+target = "aws-kms-key-arn.txt#section-2.5"
```

Comment:
Why did CI miss this earlier?

Reply:
Never mind, same root cause I'm fixing in #17. Copying this over there to unblock my own PR.

Reply:
Duvet was updated